src/com/pentalog/pentastagiu/homework3/ex3/User.java

```diff
@@ -0,0 +1,28 @@
+package com.pentalog.pentastagiu.homework3.ex3;
+
+public class User {
+    private String username;
+    private String email;
+
+    public User(String user, String email) {
+        super();
+        this.username = user;
+        this.email = email;
+    }
+
+    private void sendMessage(String message) {
+        System.out.println("From: " + this.username + "\nText: " + message + "\n");
+    }
+
+    public static void main(String[] args) {
+        User user1 = new User("user1", "user1@yahoo.com");
+        User user2 = new User("user2", "user2@yahoo.com");
+        User user3 = new User("user3", "user3@yahoo.com");
+
+        user1.sendMessage("aaaa");
```

Comment:
ok, dar ai ales un caz simplu de test; poti sa te gandesti si sa mai evoluezi codul pentru a permite diverse scenarii : sa ai in Board o lista de mesaje(fiecare sa aiba un user asociat) si apoi sa ai diverse metode: afisarea mesajelor de la userul cu emailul x, afisarea mesajelor postate in ultima ora(asta daca un mesaj ar avea si o data asociata) - sunt doar sugestii de scenarii la care te poti gandi daca vrei sa evoluezi implementarea curenta, care acopera un caz static, in care printezi secvential niste informatii la consola